_data/fr/general.yml

```diff
@@ -0,0 +1,13 @@
+footer:
+  copyright: Logiciel développé par <a href="https://github.com/corrados" target="_blank" rel="noreferrer">Volker Fischer</a> et les <a href="https://github.com/corrados/jamulus/graphs/contributors" target="_blank" rel="noreferrer">contributeur·trice·s</a>
+  alt:
+    github:          "Dépôt Jamulus sur GitHub"
+    facebook:        "Groupe Facebook Jamulus"
+    sourceforge:     "Optenez Jamulus, Internet Jam Session Software, sur SourceForge.net. Téléchargement rapide, sûr et gratuit de logiciels libres et à code source ouvert."
+btnEditAlt:          "Editer ce site"
+tNoJSEnabled:        "Vous avez désactivé ou votre navigateur ne prend pas en charge le JavaScript. C'est très bien. Ce site a été conçu pour fournir des fonctionnalités de base sans Javascript. L'activation de celui-ci peut cependant vous donner des fonctionnalités supplémentaires sur ce site."
```

Comment:
"Vous avez désactivé ou votre navigateur ne prend pas en charge le JavaScript." -> strange and unusual formulation. Probably "Vous avez désactivé le JavaScript ou votre navigateur ne prend pas en charge".

That said, your proposal here is not a correct translation. You are adding/inventing things here. A better translation is: "JavaScript est désactivé dans votre navigateur", not "Vous avez désactivé ou votre navigateur ne prend pas en charge le JavaScript". If you think this is an important clarification, it would be best to discuss/PR an improvement to the English version first. This would benefit all languages and make it easier to keep translations synchronised.

Reply:
Oui, c'est volontaire.
Le bête et laconique « JavaScript est désactivé dans votre navigateur » qu'on peut voir dans le texte indexé par les moteurs de recherche m'agace. ;)
C'est peut être mal formulé. Mais ce n'est pas parce que la version anglaise est laconique que la version française doit l'être aussi.

Aujourd'hui soit le JavaScript (et très peu de chose justifie son utilisation sur cette documentation) est non supporté parce que navigateur textuel ou synthèse vocale utilisé par les malvoyant, soit robot d'indexation, soit démarche *volontaire* de l'internaute. D'où ma formulation. (il y manquerait deux virgules pour être correcte en l'état).

/cc @Saimyx

Reply:
> C'est peut être mal formulé. Mais ce n'est pas parce que la version anglaise est laconique que la version française doit l'être aussi.

EN translation. (please @gegeweb, keep the discussion in English, hence original writers can understand what we're discussing)
> It may be poorly formulated. But just because the English version is laconic does not mean that the French version should be too.

On the contrary, we do translation. When there are several translators, if everyone goes for their own personal interpretation, it's a recipe for disaster, and for long-term non-maintainability. The translation must respect the author's original idea. If someone thinks it could be improved, then the original version should be told first.